generator/generator-opc-generator/generators/app/templates/src/_style.scss

```diff
@@ -0,0 +1,3 @@
+p {
+  font-size: var(--op-component--text-size, 16px);
```

Comment:
u might need to add the styles on :host instead of p tag

Reply:
Removed the styles for now